### PR TITLE
Sign up

### PR DESCRIPTION
**Pull request by levanhuan**

### CODE REVIEW

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<% provide(:title, 'Sign up') %>
+<h1>Sign up</h1>
```
Comment (phannh58) [inline app/views/users/new.html.erb]: i18n

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      <%= f.label :email %>
+      <%= f.email_field :email, class: 'form-control' %>
+
+      <%= f.label :password %>
+      <%= f.password_field :password, class: 'form-control' %>
+
+      <%= f.label :password_confirmation, "Confirmation" %>
+      <%= f.password_field :password_confirmation, class: 'form-control' %>
+
+      <%= f.submit "Create my account", class: "btn btn-primary" %>
```
Comment (phannh58) [inline app/views/users/new.html.erb]: i18n

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1,9 +1,22 @@
...
+      <%= f.label :password %>
+      <%= f.password_field :password %>
+
+      <%= f.label :password_confirmation, "Confirmation" %>
+      <%= f.password_field :password_confirmation %>
+
+      <%= f.submit "Create my account", class: "btn btn-primary" %>
+    <% end %>
+  </div>
 </div>
```
Comment (phannh58) [inline app/views/users/show.html.erb]: anh oi cai show nay sao lai giong cai file sign up the, show nhiem vu cua no la show ra chi tiet cac thong tin ve 1 user ma

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -6,4 +6,19 @@ def show
...
     @user = User.new
   end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = "Welcome to the book review system"
+      redirect_to @user
+    else
+      render 'new'
```
Comment (phannh58) [inline app/controllers/users_controller.rb]: anh dùng double quote

File: app/views/users/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<% provide(:title, t("signup.signup_button")) %>
+<h1> <%= t "signup.signup_button" %></h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for (@user) do |f| %>
```
Comment (phannh58) [inline app/views/users/new.html.erb]: remove ()

File: app/views/users/new.html.erb [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+      <%= render 'shared/error_messages', object: @user %>
```
Comment (phannh58) [inline app/views/users/new.html.erb]: dùng double quote nha anh

File: app/views/users/new.html.erb [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+      <%= f.label :name %>
+      <%= f.text_field :name, class: 'form-control' %>
```
Comment (phannh58) [inline app/views/users/new.html.erb]: double quote

File: app/views/users/new.html.erb [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+      <%= f.label :email %>
+      <%= f.email_field :email, class: 'form-control' %>
```
Comment (phannh58) [inline app/views/users/new.html.erb]: double quote

File: app/views/users/new.html.erb [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+      <%= f.label :password %>
+      <%= f.password_field :password, class: 'form-control' %>
```
Comment (phannh58) [inline app/views/users/new.html.erb]: double quote

File: app/views/users/new.html.erb [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+      <%= f.label :password_confirmation %>
+      <%= f.password_field :password_confirmation, class: 'form-control' %>
```
Comment (phannh58) [inline app/views/users/new.html.erb]: double quote

File: app/assets/stylesheets/custom.scss [hunk 11]
```diff
@@ -142,3 +142,31 @@ span.picture {
     border: 0;
   }
 }
+/* forms */
```
Comment (levanhuan) [inline app/assets/stylesheets/custom.scss]: xoa comment

File: app/controllers/users_controller.rb [hunk 12]
```diff
@@ -6,4 +6,19 @@ def show
   def new
     @user = User.new
   end
+
+  def create
+    @user = User.new(user_params)
```
Comment (levanhuan) [inline app/controllers/users_controller.rb]: remove ()

File: app/controllers/users_controller.rb [hunk 13]
```diff
@@ -6,4 +6,19 @@ def show
...
+    if @user.save
+      flash[:success] = "Welcome to the book review system"
```
Comment (levanhuan) [inline app/controllers/users_controller.rb]: i18n

File: app/controllers/users_controller.rb [hunk 14]
```diff
@@ -6,4 +6,19 @@ def show
...
+      redirect_to @user
+    else
+      render "new"
```
Comment (levanhuan) [inline app/controllers/users_controller.rb]: render :new

File: app/views/users/new.html.erb [hunk 15]
```diff
@@ -0,0 +1,24 @@
+<% provide(:title, t("signup.signup_button") ) %>
```
Comment (levanhuan) [inline app/views/users/new.html.erb]: space

**Comment by k-i-k-ichi** [comment]
ok

**Pull request merged**